### PR TITLE
PMF classification fix

### LINKED ISSUE #2898: PMF can output Classification of 0
I would expect that PMF (like SMRF) would classify **all points** as either 1 (unclassified) or 2 (ground), but I've seen several examples lately, where some number of points retain the classification value 0 (created, never classified). This should only be possible if some points are ignored, either through the ignore option or through return filtering.

### PR DESCRIPTION
**Pull request by chambbj**
- Do not touch classification of points exluded by return information

- Initialize all points passed to processGround() by setting
  classification to ClassLabel::Unclassified

- Apply clang-format

Fix #2898

### CODE REVIEW

**Comment by abellgithub** [comment]
Shouldn't all the points in the second group be marked as Unclassified, too?  Should we just classify everything as Unclassified?

**Comment by chambbj** [comment]
@abellgithub I debated this, and could still be convinced, but my rationale is this. Any points excluded from processing, either by `ignore` or return filtering, should go untouched, preserving their incoming classification tags. I'm certain that's the right thing to do for `ignore`, because those were very likely tagged as noise. For return filtering, I could probably go either way honestly.

**Comment by abellgithub** [comment]
I'm OK with whatever, but I think it needs clear documentation in the code.

**Comment by chambbj** [comment]
@abellgithub I added some comments and better names for the various PointViews. Also realized the same thing could've happened in SMRF, just had never been observed.

File: test/unit/filters/PMFFilterTest.cpp [hunk 1]
```diff
@@ -60,3 +63,42 @@ TEST(PMFFilterTest, validReturns)
     PointTable table;
     EXPECT_NO_THROW(filter.prepare(table));
 }
+
+TEST(PMFFilterTEst, classOneAndTwoOnly)
```
Comment (abellgithub) [inline test/unit/filters/PMFFilterTest.cpp]: Capitalization typo in name.

**Comment by chambbj** [comment]
@abellgithub If you are talking whitespace and newlines, it's just because I ran `clang-format` which is my typical go-to. If you'd like, I could back them out. If you are talking variable renaming, I'd argue to keep those - the original names were bad IMO.

**Pull request merged**